authors/apps/articles/tests/Basetest.py

```diff
@@ -36,7 +36,8 @@ def setUp(self):
             "tag_list": ["dragons", "fantacy"]
         }
         self.article_with_missing_fields = {
-            "title": "Getting started with JS"
+            "title": "Getting started with JS",
```

Comment:
Add tests for readtime

Reply:
> @dnuwa I see you're returning a string as the read time. What if it's more than 60 minutes?

The read time to all articles is returned in minutes even when it's beyond 60.

Reply:
> Add tests for read_time use a random string generator or faker library to generate body

A faker library has been used to cater for this.